### PR TITLE
Manual cleanup

### PR DESCRIPTION
**Pull request by farodin91**
* Remove comment out stuff (mostly over 6 years comment out)
* Remove unused classes

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there an issue associated with this PR? Is it referenced in the commit message?
- [ ] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [ ] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?

### CODE REVIEW

File: janusgraph-core/src/main/java/org/janusgraph/graphdb/olap/QueryContainer.java [hunk 1]
```diff
@@ -118,14 +112,7 @@ private QueryBuilder() {
...
 
-        private Query relations(RelationCategory returnType) {
-//            if (name==null) {
-//                if (hasSingleType()) name = getSingleType().name();
-//                else if (!requiresName) name = QUERY_NAME_PREFIX + queries.size();
-//                else throw new IllegalStateException("Need to specify an explicit name for this query");
-//            }
-
-            BaseVertexCentricQuery vq = super.constructQuery(returnType);
+        private Query relations(RelationCategory returnType) {            BaseVertexCentricQuery vq = super.constructQuery(returnType);
```
Comment (rngcntr) [inline janusgraph-core/src/main/java/org/janusgraph/graphdb/olap/QueryContainer.java:115]: Split this line, please

**Review by rngcntr — CHANGES_REQUESTED** [review]

File: janusgraph-core/src/main/java/org/janusgraph/graphdb/olap/QueryContainer.java (hunk 1 above)
Comment (farodin91) [reply janusgraph-core/src/main/java/org/janusgraph/graphdb/olap/QueryContainer.java:115]: good catch

Comment (rngcntr) [reply janusgraph-core/src/main/java/org/janusgraph/graphdb/olap/QueryContainer.java:115]: +1 -1920 lines
The +1 made me curious :D

File: janusgraph-core/src/main/java/org/janusgraph/graphdb/util/AllEdgesIterable.java [hunk 2]
```diff
@@ -1,92 +0,0 @@
...
-
-/**
- * Constructs {@link Iterable}s over all {@link org.janusgraph.core.JanusGraphEdge}s connecting a provided set of vertices.
- * <p>
- * Given a set of vertices, one may be interested in all edges that are contained in the subgraph spanned
- * by those vertices.
- *
- * @author Matthias Br&ouml;cheler (me@matthiasb.com);
- */
-public class AllEdgesIterable {
```
Comment (li-boxuan) [inline janusgraph-core/src/main/java/org/janusgraph/graphdb/util/AllEdgesIterable.java:31]: I think the AllEdgesIterable and other classes under this package might be used by users directly.

Comment (li-boxuan) [reply janusgraph-core/src/main/java/org/janusgraph/graphdb/util/AllEdgesIterable.java:31]: Should we mark them as deprecated in this release and maybe remove them in the next one?

Comment (farodin91) [reply janusgraph-core/src/main/java/org/janusgraph/graphdb/util/AllEdgesIterable.java:31]: I think it is a good idea to deprecated them. They are not maintained over years, not tested and not documented.

Comment (farodin91) [reply janusgraph-core/src/main/java/org/janusgraph/graphdb/util/AllEdgesIterable.java:31]: classes are deprecated

**Review by rngcntr — APPROVED** [review]

**Pull request merged**